app/vip-config/__helpers.php

```diff
@@ -0,0 +1,431 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inpsyde\Vip;
+
+const REDIRECT_STATUS_CODES = [
+    300 => 'Multiple Choices',
+    301 => 'Moved Permanently',
+    302 => 'Found',
+    303 => 'See Other',
+    304 => 'Not Modified',
+    305 => 'Use Proxy',
+    306 => 'Reserved',
+    307 => 'Temporary Redirect',
+    308 => 'Permanent Redirect',
+];
+
+const ENV_NORMALIZATION_MAP = [
```

Comment:
Can we also add a few more custom (per need) here or should we introduce the possibility to manage them through `vip-config` ?

Right now I only see 1 difference which is `'training' => 'training'`.

Reply:
Very custom things like those (that is clinet-specific) are garder to get standardized.
You ve the possibility to define `WP_ENVIRONMENT_TYPE` upfront t ohandle thiscase.
But also, an environment like "training" is something that you would likee to map to `staging`, because it is not producion, nor `development`. And the default value for the funcion is `staging` so this would work already :)

Reply:
Problem here is with the constants and like setting the cookie domain, which than uses the one from staging. Hmm.